rickandmorty/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "rickandmorty",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.5.0",
+    "@testing-library/user-event": "^7.2.1",
+    "bootstrap": "^4.5.0",
+    "react": "^16.13.1"
```

Comment:
Not sure what happened here, but the dependencies list is missing `react-dom`, `react-router-dom`, and `react-scripts`